Sources/Polyfill.swift

```diff
@@ -0,0 +1,118 @@
+import Foundation
+
+extension Date {
+  func _ISO8601Format() -> String {
+    if #available(iOS 15.0, macOS 12.0, *) {
+      return ISO8601Format()
+    } else {
+      return ISO8601DateFormatter.string(from: self,
+                                         timeZone: TimeZone(secondsFromGMT: 0)!,
+                                         formatOptions: [.withInternetDateTime])
+    }
+  }
+}
+
+struct _AsyncBytes: AsyncSequence {
```

Comment:
Likewise for `AsyncBytes` regarding the `@available(…)` attribute comments above.